pkg/bucketaccessrequest/bucketaccessrequest.go

```diff
@@ -131,6 +143,10 @@ func (b *bucketAccessRequestListener) provisionBucketAccess(ctx context.Context,
 		return err
 	}
 
+	if !util.CheckFinalizer(bucketAccessRequest, util.BARDeleteFinalizer) {
```

Comment:
In [L142](https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/29/files#diff-967e432aed6a8d88ba1f533d2aa548e67f3cc310d734ccdb66deeeab2aaef788R142) above what about the case of the BA already existing? This would be true in brownfield access.

Reply:
brownfield case is not coded yet but my understanding is that brownfield case Bucket exist but BA/BAR works the same.

Reply:
The BA was just created so why do we need to check for the delete finalizer here? `CheckFInalizer` doesn't re-Get the object so we are dealing with the in-memory BA that was just created.